src/civ2.js

```diff
@@ -52,183 +65,148 @@ module.exports = function(robot) {
         const tagTxt = tag ? `tag ${tag}` : 'default tag';
         msg.reply(`The deployment of ${tagTxt} to ${target.trigger} is scheduled.`);
       } catch (ex) {
-        console.error(ex);
+        error(ex);
         msg.reply(`Sorry, something went wrong: ${ex.message}`);
       }
     });
   }
 
-  robot.hear(/release stoic (\S*)/, async (msg) => {
-    const tag = msg.match[1];
-    try {
+  robot.hear(
+    /release stoic (\S*)/,
```

Comment:
With `*` instead of `+`? Is the tag optional? In that case, `(\S+)?` would better document that.

Reply:
Also, you should use flag `i` for case-insisitiveness. Let me write my orders by starting with a capital, I like my sentences to be correct English :D That will also let me shout to the bot in ALL CAPS, when I'm in the mood!